src/shim.js

```diff
@@ -91,23 +91,39 @@ const commands = {
         completeCommand(command);
     },
 
+    loadPageContent: function (command) {
+        let page = objectSpace[command.target];
+        if(!page) {
```

Comment:
Hmm interesting. But if you call `page.render` and `render` fails, then I agree. It should return false. But if can't find the page in `objectSpace`, that's probably a bug. That's probably more serious because `render` didn't fail. Shim.js failed 👎 probably not good